contribute.md

```diff
@@ -232,7 +235,7 @@ Here are a few rules to keep in mind while writing a commit message
 
  A good commit message looks something like this
 ```
- Summarize changes in around 50 characters or less
+ <type>: Summarize changes in around 50 characters or less
```

Comment:
Ready

Reply:
Every type is going to have to be mapped to a `MAJOR`, `MINOR`, or `PATCH` semantic version in our tooling, and also to whether it goes into the `New Feature`, `Bug Fix` or `Other Changes` section of the CHANGELOG.  For example - does `chore` always mean `PATCH` and `Other Changes`?  I think in most cases the use of types is pretty obvious - for example, if I change something in `.github` that will be a type `ci` (`PATCH`, `Other Changes`), if I'm fixing `ansible-lint` issues that will probably be type `style` (`PATCH`, `Other Changes`), changes to something in `tests/` will be type `test` (`PATCH`, `Other Changes`).  I'm thinking a `perf` change is equivalent to a bug fix (`PATCH`, `Bug Fix`) - that is, poor performance is a "bug".  We should provide some sort of guidance for system roles developers (and ourselves!) so that it is easy to know which type to apply to a given situation, and how that will be translated into the changelog.  e.g. "Use `docs(changelog): ..` for updates to CHANGELOG.md"

Reply:
My understanding is

- `<type>!:` with exclamation mark, which stand for BREAKING CHANGE always bumps `MAJOR`
  Conventional commits allow for adding either exclamation mark like above, or BREAKING CHANGE to footer, but I'd like to keep it simple and only allow one variant - `!`.
- `feat:` and `fix:` types bump `MINOR`
- Any other types bump `PATCH`. And it can be pretty much anything, I don't even think that we must limit this. In addition to what you listed there can be `performance`, `test` etc.

Reply:
I think I covered everything in my last commit